Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -3933,6 +3933,19 @@ InspectorStartsAttached:
     WebKit:
       default: true
 
+InspectorStoreCertificateInfo:
```

Comment:
instead of `*Store*` i think we should make this more generic like `InspectorSupportsShowingCertificate` and use it inside `InspectorFrontendHost::supportsShowCertificate` too.  this way, everything is connected by a single adjustable preference instead of hardcoded `#if`